mobile/src/navigations/onboarding-navigation/components/step/step.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties = {
+    index: number;
+    isFocused: boolean;
+    routeName: string;
+    onPress: () => void;
+};
+
+const Step: React.FC<Properties> = ({
```

Comment:
there should be completed, disabled options 
<img width="222" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/103518438/535ebf5c-b842-480f-83c7-bc39912d934f">

Reply:
I am a bit stuck with it, as I understood I should pass this two props (isCompleted and disabled) through options in Drawer.Screen. 
`<Drawer.Screen
                name={OnboardingScreenName.CV_AND_CONTACTS}
                component={CvAndContacts}
                options={{ isCompleted: false, disabled: false }}
            />`
These options are succesfully passed and I can extract them, but I can't find a way how to type them and constantly get an error that they are not assignable to type `DrawerNavigationOptions`. Do you have any suggestions how I can solve this issue?

Reply:
Solved, passed props through initialParams (I don't why I thought that I should pass it through options)